provider/server/machine_create_test.go

```diff
@@ -219,15 +202,10 @@ var _ = Describe("CreateMachine", func() {
 		By("creating a machine with boot image and single empty disk")
 		createResp, err := machineClient.CreateMachine(ctx, &iri.CreateMachineRequest{
 			Machine: &iri.Machine{
-				Metadata: &irimeta.ObjectMetadata{
-					Labels: map[string]string{
-						"machinepoolletv1alpha1.MachineUIDLabel": "foobar",
-					},
-				},
+				Metadata: &irimeta.ObjectMetadata{},
 				Spec: &iri.MachineSpec{
-					Power: iri.Power_POWER_ON,
 					Image: &iri.ImageSpec{
-						Image: osImage,
+						Image: squashfsOSImage,
 					},
 					Class:        machineClassx3xlarge,
 					IgnitionData: ignitionData,
```

Comment:
ignition data aren't valid. I recommend delete them because it can impact boot of VM

Reply:
removed